ServerSide/yoty/WebApi/Controllers/SuppliersController.cs

```diff
@@ -44,6 +44,23 @@ public async Task<ActionResult<SupplierDTO>> GetSupplier(string supplierId)
             return this.StatusCode(StatusCodes.Status404NotFound, response.SuccessOrFailureMessage);
         }
 
+        [HttpGet]
+        [Route("{supplierId}/bids")]
+        public async Task<ActionResult<BidsDTO>> GetSupplierBids(string supplierId, [FromQuery] SupplierBidsRequestOptions supplierBidsRequestOptions)
```

Comment:
do we want to make it authorize call or public ?

Reply:
It's for the personal area, not sure what 'authorize call' means but not public